### PR TITLE
Add sentry-native/0.4.17

### PR DESCRIPTION
**Pull request by MartinDelille**
Specify library name and version:  **sentry-native/0.4.17**

---

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [ ] I've tried at least one configuration locally with the [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Comment by MartinDelille** [comment]
Waiting for https://github.com/conan-io/conan-center-index/pull/10428 and https://github.com/conan-io/conan-center-index/pull/10429

**Comment by MartinDelille** [comment]
@uilianries @madebr If I understand well the new policy, your two positive review should be enough to get this merged!

**Comment by uilianries** [comment]
> @uilianries @madebr If I understand well the new policy, your two positive review should be enough to get this merged!

Correct.

File: recipes/sentry-native/all/conanfile.py [hunk 1]
```diff
@@ -36,11 +36,9 @@ class SentryNativeConan(ConanFile):
         "qt": False,
         "with_crashpad": "sentry",
         "with_breakpad": "sentry",
-        "performance": False,
```
Comment (uilianries) [inline recipes/sentry-native/all/conanfile.py:39]: Why not implement, instead of removing?

Comment (MartinDelille) [reply recipes/sentry-native/all/conanfile.py]: Ok done here: [6b2d284](https://github.com/conan-io/conan-center-index/pull/10439/commits/6b2d284115eba93c4cd8aaa069ebed9e3c3ac472)

**Review by madebr — APPROVED** [review]

**Review by ericriff — APPROVED** [review]

**Review by SSE4 — APPROVED** [review]

**Pull request merged**